src/utils/lib.js

```diff
@@ -8,8 +8,21 @@ export const WINDOW_INNER_WIDTH = window.innerWidth;
 export const scaleMultiplier = WINDOW_INNER_HEIGHT / 768.0;
 export const fullscreenScaleMultiplier = 136;
 
-export const isiOS = (typeof AndroidInterface == 'undefined');
-export const isAndroid = (typeof AndroidInterface != 'undefined');
+export function detectOS () {
+    var userAgent = window.navigator.userAgent.toLowerCase();
+    const ios = /iphone|ipod|ipad/.test( userAgent );
+    // safari = /safari/.test( userAgent ), // currently do not need to detect browser vs webview
+    // android = /android/.text.(userAgent);
+
+    if( ios ) {
```

Comment:
I think this would be more consistent with our style:
```suggestion
    if (ios) {
```